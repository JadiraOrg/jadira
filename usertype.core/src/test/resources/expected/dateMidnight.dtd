<!ELEMENT dataset (
    DATEMIDNIGHT*)>

<!ELEMENT DATEMIDNIGHT EMPTY>
<!ATTLIST DATEMIDNIGHT
    ID CDATA #REQUIRED
    MY_DATE CDATA #IMPLIED
    MY_OFFSET CDATA #IMPLIED
    NAME CDATA #IMPLIED
>

