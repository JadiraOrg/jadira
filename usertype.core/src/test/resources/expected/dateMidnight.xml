<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE dataset SYSTEM "dateMidnight.dtd">
<dataset>
  <DATEMIDNIGHT ID="0" MY_DATE="2004-02-25" MY_OFFSET="+04:00" NAME="test_0"/>
  <DATEMIDNIGHT ID="1" MY_DATE="1980-03-11" MY_OFFSET="UTC" NAME="test_1"/>
  <DATEMIDNIGHT ID="2" NAME="test_2"/>
</dataset>